tests/test_other.py

```diff
@@ -617,7 +617,11 @@ def test_emcc_print_search_dirs(self):
     output = self.run_process([EMCC, '-print-search-dirs'], stdout=PIPE).stdout
     self.assertContained('programs: =', output)
     self.assertContained('libraries: =', output)
-    self.assertContained(str(shared.Cache.get_lib_dir(absolute=True)), output)
+    libpath = output.split('libraries: =', 1)[1].strip()
+    libpath = libpath.split(os.pathsep)
+    libpath = [Path(p) for p in libpath]
+    print(libpath)
+    self.assertIn(shared.Cache.get_lib_dir(absolute=True), libpath)
```

Comment:
so this is now asserting that the lib dir is contained in one of the split components of the path?

Reply:
Yes, its more precise.. and it parse the paths via the `Path` constructor above so any slashes get normalized automatically (which was the reason for the failure).

Reply:
ok, can you say this in the commit description?